dashboard/app/models/script_level.rb

```diff
@@ -66,7 +66,8 @@ def oldest_active_level
 
   def active?(level)
     properties_hash = JSON.parse(properties)
-    !properties_hash[level.name] || properties_hash[level.name]['active'] != false
+    variants = properties_hash['variants']
+    !variants || !variants[level.name] || variants[level.name]['active'] != false
```

Comment:
Doing it like that treats `nil` like `false`, but I wanted active to default to `true` if it wasn't specified.

It shouldn't matter though, because I don't think there's any way to end up with a non-nil `variants[level.name]` but a nil `variants[level.name][active]`, and there are certainly no scriptlevels like that at the moment. Using `variants['level.name']['active']` (without the `!`) would be fine.